sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/AesKw.java

```diff
@@ -134,7 +135,8 @@ public ICryptoTransform createEncryptor(byte[] key, byte[] iv, Provider provider
         if (iv != null) {
             // iv length must be 64 bits
             if (iv.length != 8) {
-                throw logger.logExceptionAsError(new IllegalArgumentException("iv length must be 64 bits"));
+                throw logger.logExceptionAsError(new IllegalArgumentException(String.format(
+                    "iv length must be %s bits", BLOCK_SIZE_IN_BITS)));
```

Comment:
You've made it possible here for the block size to be changed in a central place, and it feels like you're introducing a potential future bug by still having the line `if (iv.length != 8) {`. Perhaps 64 will never change and this change is needless, but if you keep the code like this I would recommend either externalising the `8`, or perhaps more appropriately, documenting alongside the new static field the requirement that if this value changes that future developers must update the conditionals everywhere appropriately.

Put another way, what motivated you to extract the value 64 out like this? I probably wouldn't have done it.

Reply:
To be completely honest,I think when I was first writing this I saw somewhere else a similar piece of code with a hard-coded value, based this off of that and then forgot to change it into a static field at a later point. Your suggestion is what I would prefer to do as well by default. Thanks for pointing it out :)

Reply:
I took a second look at this class and it seems the hard-coded value was there already, I will change it to what you mention.